dynamiqs/time_array.py

```diff
@@ -302,15 +306,18 @@ def shape(self) -> tuple[int, ...]:
     def mT(self) -> TimeArray:
         return ConstantTimeArray(self.array.mT)
 
-    def __call__(self, t: ScalarLike) -> Array:  # noqa: ARG002
-        return self.array
-
```

Comment:
Could you re-order the method `__call__` as before in all subclasses? (to see the changes in git + to match base class ordering).

Reply:
The base class has `__call__` after all non `__` methods, hence the reordering